_rules/sequentially-focusable-element-has-visible-focus-oj04fd.md

```diff
@@ -0,0 +1,232 @@
+<span tabindex="0">Act rules</span>
+```
+
+#### Passed Example 2
+
+The first [focusable][] element is part of [sequential focus navigation]. The set of pixels that change color when it is [focused][] (due to default User Agent's styling) is unique. The second [focusable][] element is not applicable because it has been removed from [sequential focus navigation][] due to the `tabindex` attribute. Its presence is nonetheless enough to make the first one applicable.
+
+```html
+<a href="https://act-rules.github.io/">ACT rules</a> <button tabindex="-1">Dummy button</button>
+```
+
+#### Passed Example 3
+
+The first [focusable][] element, is part of [sequential focus navigation][]. The set of pixels that change color when it is [focused][] (the border of the `span` element) is unique (it does not change when any other element is [focused][]).
+
+```html
+<link rel="stylesheet" href="../test-assets/focus-visible/styles.css" />
+<script src="../test-assets/focus-visible/script.js" />
+
+<span id="indicator" class="border">
+	<a
+		id="act"
+		class="no-focus-default"
+		onfocus="toggleActivation('indicator')"
+		onblur="toggleActivation('indicator')"
+		href="https://act-rules.github.io/"
+		>ACT rules</a
+	>
+</span>
+<button>Dummy button</button>
+```
+
+#### Passed Example 4
+
+For each of these three [focusable][] elements, the set of pixels changing color when it is [focused][] is unique. For the element with `id` "wcag", each of these pixels also change color when another element is [focused][], but they do not all change color for the same other element. Thus, the set of pixels changing color is also unique for this one.
+
+```html
+<link rel="stylesheet" href="../test-assets/focus-visible/styles.css" />
+<script src="../test-assets/focus-visible/script.js" />
+
+<span id="indicator-act" class="indicator solid"></span>
+<a
+	id="act"
+	class="no-focus-default"
+	onfocus="toggleActivation('indicator-act'); toggleActivation('indicator-wcag')"
+	onblur="toggleActivation('indicator-act'); toggleActivation('indicator-wcag')"
+	href="https://act-rules.github.io/"
+	>ACT rules</a
+>
+<span id="indicator-wcag" class="indicator solid"></span>
+<a
+	id="wcag"
+	class="no-focus-default"
+	onfocus="toggleActivation('indicator-wcag'); toggleActivation('indicator-w3c')"
+	onblur="toggleActivation('indicator-wcag'); toggleActivation('indicator-w3c')"
+	href="https://www.w3.org/TR/WCAG21/"
+	>WCAG</a
+>
+<span id="indicator-w3c" class="indicator solid"></span>
+<a
+	id="w3c"
+	class="no-focus-default"
+	onfocus="toggleActivation('indicator-w3c'); toggleActivation('indicator-final')"
+	onblur="toggleActivation('indicator-w3c'); toggleActivation('indicator-final')"
+	href="https://www.w3.org/"
+	>WCAG</a
+>
+<span id="indicator-final" class="indicator solid"></span>
+```
+
+### Failed
+
+#### Failed Example 1
+
+None of these [focusable][] elements have any pixel changing color when they are [focused][] because the default styling has been overwritten.
```

Comment:
```suggestion
None of these [focusable][] elements have any pixel changing color when they are [focused][] because the default styling has been overwritten by a style that removes the outline.
```